spec/models/event_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe Event, type: :model do
+
+  let!(:event){ create(:event) }
```

Comment:
Зачем с восклицательным знаком?

Reply:
И я не очень уверен, что за ивент/юзер тут у тебя, но если они должны быть связаны, то надо это явно указать:
```suggestion
  let(:event){ create(:event, user: user) }
```